frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/EditFieldDrawer.tsx

```diff
@@ -150,30 +106,27 @@ export const EditFieldDrawer = (): JSX.Element | null => {
 
 interface MemoFieldDrawerContentProps {
   field: FieldCreateDto
-  isLoading: boolean
-  isPendingField: boolean
-  handleChange: (field: FieldCreateDto) => void
-  handleSave: (field: FieldCreateDto, options?: FieldMutateOptions) => void
-  handleCancel: () => void
 }
 
-const MemoFieldDrawerContent = memo((props: MemoFieldDrawerContentProps) => {
-  // Extract field variable just to get field.fieldType types to cooperate
-  const field = useMemo(() => props.field, [props.field])
-  switch (field.fieldType) {
-    case BasicField.Checkbox:
-      return <EditCheckbox {...props} field={field} />
-    case BasicField.Nric:
-      return <EditNric {...props} field={field} />
-    case BasicField.Section:
-      return <EditHeader {...props} field={field} />
-    case BasicField.Uen:
-      return <EditUen {...props} field={field} />
-    case BasicField.YesNo:
-      return <EditYesNo {...props} field={field} />
-    case BasicField.Radio:
-      return <EditRadio {...props} field={field} />
-    default:
-      return <div>TODO: Insert field options here</div>
-  }
-})
+export const MemoFieldDrawerContent = memo(
+  (props: MemoFieldDrawerContentProps) => {
+    // Extract field variable just to get field.fieldType types to cooperate
+    const field = useMemo(() => props.field, [props.field])
```

Comment:
Not from this PR, but... this useMemo()... 🙄 😑

Reply:
removed the `useMemo` in 11533f48 after reading [this article](https://kentcdodds.com/blog/usememo-and-usecallback)!

Reply:
Thanks for that 🙏 !

I'm scheduling an item for a short eng discussion on this topic tomorrow.